test/test_sparse.py

```diff
@@ -517,6 +520,27 @@ def _test_spadd_shape(self, shape_i, shape_v=None):
 
         self.assertEqual(res, expected)
 
+        x, i, v = self._gen_sparse(len(shape_i), 10, shape)
+        nnz = i.size(1)
+
+        # Non contiguous sparse indices tensor
```

Comment:
I'm not convinced that we actually care about supporting this case, but we can make a further determination on this as we discover how much work it is to keep this working.